share/smack/versions.py

```diff
@@ -0,0 +1 @@
+VERSIONS = {'RUST_VERSION': 'nightly-2020-05-15'}
```

Comment:
Wait, why is this here if we already have the `bin/versions` file?
I would really prefer to have to update versions in only one place.

Reply:
I proposed a solution here: https://github.com/smackers/smack/compare/develop...add-llvm-version?expand=1